### ISSUE TITLE
Keyboard shortcut for switching focus to a different monitor

### ISSUE THREAD

**Issue by lane-s** [linuxmint/cinnamon#7860]
We already have the ability to move windows between monitors and workspaces using the keyboard, but I would also like the ability to change the focused monitor with a keyboard shortcut.

Sometimes I have an application that is controlled primarily by keyboard shortcuts up on each monitor, so reaching for the mouse just to focus the other application always feels like an interruption.

**Comment by germanfr** [comment]
Hey. You can change the focused window with <kbd>Alt</kbd>+<kbd>TAB</kbd>.

**Comment by icarter09** [comment]
@lane-s did @germanfr comment help with your issue?

**Comment by lane-s** [comment]
> @lane-s did @germanfr comment help with your issue?

To be honest, I can't remember quite what I was thinking when I wrote this, and I've since switched to using a tiling window manager. I definitely knew about alt-tab at the time, but alt-tab isn't really monitor aware afaik. I worked around this by only using one window per monitor per workspace. 

Anyway, we can close the issue since I think the answer to people who want finer control over this stuff is probably to use a completely keyboard driven window manager.
